test/steps/test_all_steps.py

```diff
@@ -100,6 +100,9 @@ def test_step_folder(test: str, pdk_root: str):
         os.path.join(os.getcwd(), "handler.py"), "handle"
     ) or (lambda: None)
 
+    base_sdc = os.path.join(get_script_dir(), "base.sdc")
```

Comment:
A bit magical but OK.

Reply:
Better than having the file 80 times. Also it's unit tests, who cares